CONTRIBUTING.md

```diff
@@ -1,17 +1,50 @@
-### Linee guida per lo sviluppo
-
- - Board per gestire i task (TODO)
- - I task devono essere semplici e descrittivi in modo che sia chiaro il lavoro da svolgere
- - **Non** eseguire il push di commit direttamente verso il branch main
- - Implementazioni monotematiche: creare un feature branch con nome possibilmente descrittivo (es: id dell’issue + brevissima descrizione, 123-feature-foo)
- - Implementazioni articolate: creare un integration branch (es: integration/feature-foo) da cui poi forkare ulteriori feature branches specifici (es: 123-feature-foo-bar)
- - Creare pull requests più semplici possibile per facilitare la review
- - Prima di mergiare verso main (o integration branch) di seguito `destinazione`:
+# Linee guida per lo sviluppo
+
+## Indice
+
+- [**Nomenclatura dei file**](#nomenclatura-dei-file)
+- [**URL**](#url)
+- [**Workflow di lavoro**](#workflow-di-lavoro)
+
+## Workflow di lavoro
+
+### Task
+
+- Board per gestire i task (TODO)
+- I task devono essere semplici e descrittivi in modo che sia chiaro il lavoro da svolgere
+
+### GIT
+
+- **Non** eseguire il push di commit direttamente verso il branch main
```

Comment:
come nomenclatura per i feature branches io opterei per adottare la stessa convenzione usata per i branch di integrazione nel punto sotto, quindi ad esempio `feature/foo`.
Piallerei via il numero issue dal nome del branch, non sono due entità cosi fortemente legate, almeno nel nostro caso in cui abbiamo issue molto atomiche, e.g. già nella pagina di "chi siamo" io sto lavorando su due issue nello stesso branch

Reply:
L'idea era quella di tenere appunto un rapporto 1:1 tra issue e branch in realtà

Reply:
Mmmh... Parliamone, allora!

Non era così chiaro, personalmente parlando...  
Dovrò rivedere la PR #14 su cui stavo lavorando...
L'ho linkata a due Issue differenti e al medesimo branch; nel mio caso oltretutto, così come @longo-andrea, ho usato la convenzione `feature/*`! 🤔

Reply:
Sinceramente mi allineerei ad una visione meno strict, poi se la situazione è opportuna si può tranquillamente aprire un branch specifico per una issue.

Reply:
Lasciamolo pure aperto allora, nessun problema, l'importante è che le PR abbiano dimensioni facilmente revisionabili. Meglio una PR in più di una gigante.